holoviews/streams.py

```diff
@@ -392,82 +392,130 @@ def hashkey(self):
         return {'hash': uuid.uuid4().hex}
 
 
-class DataFrameStream(DataStream):
+class Buffer(Pipe):
     """
-    DataFrameStream provides a means stream pandas DataFrames. The
-    stream will accumulate a DataFrame up to the specified ``backlog``
-    of rows. The accumulated dataframe is then made available via the
+    Buffer provides a means stream and accumulate tabular datasets.
```

Comment:
a means of streaming and accumulating?  

Or maybe "Buffer allows streaming and accumulating incoming chunks of rows from tabular datasets".

Reply:
Thanks, that works better.